pkg/meta/model/job.go

```diff
@@ -862,6 +874,7 @@ func (job *Job) GetInvolvingSchemaInfo() []InvolvingSchemaInfo {
 // SubJob is a representation of one DDL schema change. A Job may contain zero
 // (when multi-schema change is not applicable) or more SubJobs.
 type SubJob struct {
+	Version     JobVersion      `json:"version"`
```

Comment:
don't add it, we only need 1 version for a job

can you change all job types of multiple schema change in this PR